closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -1040,12 +1049,16 @@ private void init(String profileID) {
                 printStackTraceAndExit(profileID + " has already been parsed. Potential infinite recursion.");
             }
 
+            loadProfileCheck(profileID);
+
             String potentialExtendsProfileID = parseProperty(securityProps.getProperty(profileID + ".extends"));
```

Comment:
Suggest a local to avoid computing `profileID + ".extends"` repeatedly:
```java
            String profileExtends = profileID + ".extends";
```

Reply:
Done.